cloud-director/v31.0.0/README.md

```diff
@@ -0,0 +1,196 @@
+
+#### Fixed
+
+- Fix linting issues.
+- Go: Update dependencies.
+
+### k8s-audit-metrics [v0.10.2...v0.10.3](https://github.com/giantswarm/k8s-audit-metrics/compare/v0.10.2...v0.10.3)
+
+#### Fixed
+
+- Fix linting issues.
+- Go: Update dependencies.
+
+### net-exporter [v1.22.0...v1.23.0](https://github.com/giantswarm/net-exporter/compare/v1.22.0...v1.23.0)
+
+#### Changed
+
+- Check for errors when closing connections.
+- Switch from Endpoints to EndpointSlices for neighbors discovery.
+
+### node-exporter [v1.20.2...v1.20.3](https://github.com/giantswarm/node-exporter-app/compare/v1.20.2...v1.20.3)
+
+#### Changed
+
+- Go: Update dependencies.
+
+### observability-bundle [v1.11.0...v2.0.0](https://github.com/giantswarm/observability-bundle/compare/v1.11.0...v2.0.0)
+
+#### Added
+
+- Add support for enabling pre-configured custom resources in KSM
+- Add metrics containing labels for Crossplane resources
+
+#### Changed
+
+- Upgrade `alloy-app` from 0.10.0 to 0.11.0
+  - This bumps the version of `Alloy` from 1.8.3 to 1.9.0
+- Upgrade `alloy-app` from 0.9.0 to 0.10.0
+  - This bumps the version of `Alloy` from 1.7.1 to 1.8.3
+- Reconfigure Flux-related part of the KSM to use wildcards instead of hardcoded versions.
+- Rename Flux-related metrics produced by the KSM.
+- Upgrade `kube-prometheus-stack` to 72.3.0
+  - Bumps prometheus-operator to 0.82.0
+  - Bumps prometheus-operator CRDs to 0.82.0
+- Upgrade `kube-prometheus-stack` to 72.3.0
+  - Bumps prometheus-operator to 0.82.0
+- Upgrade `kube-prometheus-stack` from 69.5.1 to 70.1.1
+  - Bumps prometheus-operator to 0.81.0
+  - Bumps prometheus to 3.2.1
+
+#### Fixed
+
+- Fix catalog for alloy apps as it is now pushed to the default catalog.
+
+#### Removed
+
+- Clean up old and deprecated telemetry collectors:
+  - `promtail`
+  - `grafana-agent`
+  - `promtheus-agent`
+- Disable PodSecurityPolicies by default as PodSecurityPolicies are deprecated and removed in Kubernetes v1.25+ clusters
+
+### observability-policies [v0.0.1...v0.0.2](https://github.com/giantswarm/observability-policies-app/compare/v0.0.1...v0.0.2)
+
+#### Changed
+
+- Add Cluster Role to allow latest Kyverno versions to work (https://github.com/giantswarm/giantswarm/issues/33416)
+- Switch `.Values.disabled` to `.Values.enabled` to follow best practices.
+
+### security-bundle [v1.10.1...v1.11.0](https://github.com/giantswarm/security-bundle/compare/v1.10.1...v1.11.0)
+
+#### Added
+
+- Add `policy-api-crds` app to manage Policy API CRDs.
+
+#### Changed
+
+- Update `trivy` (app) to v0.13.4.
+- Update `cloudnative-pg` (app) to v0.0.7.
+- Update `starboard-exporter` (app) to v0.8.1.
+- Update `kyverno-policy-operator` (app) to v0.0.11.
+- Update `cloudnative-pg` (app) to v0.0.9.
+
+### teleport-kube-agent [v0.10.4...v0.10.5](https://github.com/giantswarm/teleport-kube-agent-app/compare/v0.10.4...v0.10.5)
+
```

Comment:
Why is the changelog missing for Cloud Director? It's there for the other two providers. 😅